recipes/libbigwig/all/test_package/conanfile.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.build import can_run
+
+
+class TestPackageConan(ConanFile):
+    settings = "arch", "build_type", "compiler", "os"
+    generators = "CMakeDeps", "VirtualBuildEnv", "VirtualRunEnv"
+    test_type = "explicit"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def configure(self):
```

Comment:
This method is not required in the test package https://github.com/conan-io/conan-center-index/blob/dc64e77d932de43a9153c3ff6b51acb00b4ecdc1/recipes/geos/all/test_package/conanfile.py